README.md

```diff
@@ -58,6 +58,29 @@ e.g.
 `MOCK_VISITS_PRISON_ID=WWI`
 `MOCK_VISITS_VISITORS=1838,273723`
 
+### Generating APi client models
+
+For some of our external API calls we use `openapi-generator` to generate the models used in the API clients. The Open API specifications used can be found in directory `openapi-specs`.
+
+#### Updating the Open API specs
+
+Run the following commands to take a copy of the latest specs:
+
+```
+curl https://nomis-prsner-dev.aks-dev-1.studio-hosting.service.justice.gov.uk/v3/api-docs > openapi-specs/nomis-sync-api-docs.json
+curl https://activities-api-dev.prison.service.justice.gov.uk/v3/api-docs > openapi-specs/activities-api-docs.json
+```
+
+Go into the specs and reformat so they and the diffs are easier for humans to read.
```

Comment:
sounds like hard work to reformat.  I would suggest either
1. Pipe the result through `jq .`
2. Use `http` instead with `--pretty=format`

Reply:
Good shout - next time I update the specs I'll try to improve the process in this way.